modules/support-get-cluster-id.adoc

```diff
@@ -5,21 +5,31 @@
 [id="support-get-cluster-id_{context}"]
 = Obtaining your cluster ID
 
-When providing information to Red Hat Support, it is helpful to provide the unique identifier for your cluster. You can obtain your cluster ID by using the OpenShift CLI (`oc`) or by using the {product-title} web console.
+When providing information to Red Hat Support, it is helpful to provide the unique identifier for your cluster. You can have your cluster ID autofilled by using the {product-title} web console. You can also manually obtain your cluster ID by using the web console or the OpenShift CLI (`oc`).
 
 .Prerequisites
 
 * Access to the cluster as a user with the `cluster-admin` role.
-* The {product-title} CLI (`oc`) installed.
+* Access to the web console or the OpenShift CLI (`oc`) installed.
 
 .Procedure
+ifdef::openshift-enterprise,openshift-dedicated[]
+* To open a support case and have your cluster ID autofilled using the web console:
+.. From the toolbar, navigate to *(?) Help* -> *Open Support Case*.
+.. The 'Cluster ID' value is autofilled.
+endif::[]
+ifdef::openshift-origin[]
+* To open a bug and have your cluster ID autofilled using the web console:
```

Comment:
@sferich888 Should I remove "and have your cluster ID autofilled"? I just tested 4.3 prerelease, and while it opens a BZ, I don't see where it autofills a clusterID.

Reply:
No it auto populates a field that's not typically shown. It's an advanced field if I'm not mistakin

Reply:
I see it now. It isn't showing in the BZ create bug stage. Once submitted, it does autofill (along with version) in the `Environment` field.